test/support.js

```diff
@@ -107,6 +107,13 @@ givenWorkspaceFromTemplate = function(template, options, cb) {
   });
 }
 
+givenLB3Workspace = function(cb) {
+  resetWorkspace(function(err) {
+    if(err) return cb(err);
+    givenWorkspaceFromTemplate('empty-server-3.0', cb);
```

Comment:
> templates/projects/empty-server-3.0/data.js

I think it is not a good idea to add a 3.0 template, because these templates are public and thus will be listed in `apic loopback` prompt.

I think it's better to use the existing template (`empty-server`) and manually change loopback version in the scaffolded package.json.

Around the time when we will be releasing LB 3.0, we will change empty-server to scaffold LB 3.0 apps. At that point, we need to change these tests so that existing remote-method tests depending on LB 2.0 keep using LB 2.0 projects. To make that transition easier, I am proposing to add another helper `givenLB2Workspace`. Both `givenLB*Workspace` helpers should create workspace from the same template and then explicitly change loopback version in `package.json`. With those helpers in place, `describe('ModelMethod')` should be renamed to `describe('ModelMethod - LoopBack 2.x')` and call `givenLB2Workspace`.